src/lib/menu/menu.spec.ts

```diff
@@ -989,11 +1004,18 @@ describe('MdMenu', () => {
 
       expect(menus.length).toBe(3, 'Expected three open menus');
 
+      instance.rootCloseCallback.calls.reset();
```

Comment:
These reset calls shouldn't be necessary.

Reply:
They were necessary when because Observable.of(null) was triggering a close event. Replaced with Observable.empty()